tm2/pkg/bft/rpc/core/net.go

```diff
@@ -154,10 +153,14 @@ import (
 //	  }
 //
 // ```
-func NetInfo(ctx *rpctypes.Context) (*ctypes.ResultNetInfo, error) {
-	out, in, _ := p2pPeers.NumPeers()
+func NetInfo(_ *rpctypes.Context) (*ctypes.ResultNetInfo, error) {
```

Comment:
I don't understand why exactly this method, which is supposed to give information about the network, is actually updating the network information too.
Does it mean the network status is only updated upon an explicit request?